pymagicc/config.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module for collating configuration variables from various sources
+
+The order of preference is:
+Environment variable > User configuration file > Defaults
```

Comment:
does this mean it users environment variables over everything else?

Reply:
That's the normal order of precedence. The methods of setting the parameters are in order of increasing specificity and lifespan. The default parameters in code are meant to never be changed by anyone other than the module authors. The config in the user profile is static values for a given user, finally, environment variables can be set for individual shell sessions.

Reply:
Makes sense thx